tools/timedclient/timedclient.cpp

```diff
@@ -881,7 +881,7 @@ static void event_emit_details(CONST Maemo::Timed::Event &eve)
 {
   event_emit_info(eve);
 
-  printf("  MaximalTimeoutSnoozeCounter: %d\n",
+  printf("  maximalTimeoutSnoozeCounter: %d\n",
```

Comment:
I suppose this should stay upper case to be consistent with the below items. Being more kind of "pretty printed" details.

Reply:
ok